spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -348,6 +348,11 @@
         "level": "error"
       }
     },
+    {
+      "name": "spring.data.couchbase.type-key",
+      "description": "Property key used by Couchbase as the document type.",
+      "defaultValue": "_class"
+    },
```

Comment:
There's no need for additional metadata here. It will be automatically generated from the new field on `CouchbaseDataProperties`